test/src/test/java/org/locationtech/geowave/test/secondary/BasicSecondaryIndexIT.java

```diff
@@ -32,7 +32,9 @@
           GeoWaveStoreType.HBASE,
           GeoWaveStoreType.BIGTABLE,
           GeoWaveStoreType.CASSANDRA,
-          GeoWaveStoreType.DYNAMODB,
+          // TODO: DynamoDB currently disabled, because it takes abnormally long on this test
+          // (~500s)
```

Comment:
Would you be able to condense this comment into a single line?

Reply:
I would have to reword it, because the formatter forces the split there

Reply:
Done, removed "currently"